src/Widgets/Proxy/ProxyConfigurationPage.vala

```diff
@@ -212,30 +209,34 @@ namespace Network.Widgets {
 
         private void verify_applicable () {
             if (auto_button.active) {
-                apply_button.sensitive = auto_entry.text.strip () != "";
+                enable_sw->sensitive = auto_entry.text.strip () != "";
```

Comment:
Once again, just use `root_switch.sensitive !=`.

Reply:
Hello donadigo, thanks for your feedback. This was my first experience programming in Vala honestly haha, i use to code in C, by microcontrollers and so... A question, additional to all those C things, for this particular observation, it means that  `auto_entry.text.strip ()` is unnecesary, so i can delete it? I didn't do it because i prefer to not touch things i don't understand at all... so i made all changes you suggested but partially this one.